beacon-chain/rpc/eth/beacon/handlers_state.go

```diff
@@ -34,20 +34,20 @@ func (s *Server) GetStateRoot(w http.ResponseWriter, r *http.Request) {
 
 	stateId := mux.Vars(r)["state_id"]
 	if stateId == "" {
-		http2.HandleError(w, "state_id is required in URL params", http.StatusBadRequest)
+		httputil.HandleError(w, "state_id is required in URL params", http.StatusBadRequest)
```

Comment:
similar to what we did with blocker, what if we did the same thing for the other endpoints?
extract the core logic out and return response/rpcErr so we don't need a static analyzer for all these httputil.HandleError 🤔

Reply:
We would still need to remember about typing `return` after calling these functions